projectcharter.md

```diff
@@ -0,0 +1,84 @@
+# forms.gov Project Charter
+Constituents of the Federal government use some number of more than 23,000 different forms available to them to interact with agencies; these agencies use the information from these forms to provide information or benefits, but spend innumberable hours recreating and quality checking handwritten entries on the original paper forms. This method of manual form processing based on paper-only forms leads to more than 11.4 billion hours of paperwork annually. As we have seen in multiple USDS engagements, web-based forms created through user-centered design methods reduce the respondent burden while producing well-structured data to the agencies for ingestion. This project creates a foundation for optimizing the process of building more useful, modern, and frictionless web-based forms.
```

Comment:
I may not use the word "constituents", because more people than just voters interact with government forms. Maybe the way to phrase the first two sentences is: "The federal government maintains around 23,000 forms, which are used to carry out its essential functions and provide critical services and benefits."

The other point is that the problem isn't just paper forms; there are many "digital" forms that are actually just fillable pdfs, and still require manual processing because the data isn't collected in a format that can be automatically processed. I would amend the language here to be a little less focused on just paper forms, and more generally on manual data validating and processing, from both paper and digital pdf forms.

I might also add that the distinguishing factor of this library isn't it's ability to build web-based forms; there are many ways of doing that. One thing that is unique is the speed at which one can build them using this library. Our team went from building a form in 5 months to 5 days. This will save the government time and money in moving to a more modern framework. Just a thought!

Finally, I would edit the final sentence to be: "This project creates a tool for building complex web-based forms significantly faster than existing methods, using best practices in user experience and data collection, validation, and transmission."